management/src/main/java/org/ehcache/management/providers/statistics/EhcacheStatisticsProvider.java

```diff
@@ -55,6 +69,17 @@ protected void dispose(ExposedObject<CacheBinding> exposedObject) {
   }
 
   @Override
+  public final Collection<? extends Descriptor> getDescriptors() {
+    Collection<StatisticDescriptor> capabilities = new HashSet<StatisticDescriptor>();
+    for (ExposedObject o : getExposedObjects()) {
+      capabilities.addAll(((StandardEhcacheStatistics) o).getDescriptors());
+    }
+    List<StatisticDescriptor> list = new ArrayList<StatisticDescriptor>(capabilities);
+    Collections.sort(list, STATISTIC_DESCRIPTOR_COMPARATOR);
```

Comment:
Note: I forgot to apply this code before. I also did this change for server entity stats. 
This really helps during tests or in TMS/TMC when generating a json view of the backend topology with management data. Having them unsorted makes it hard to find wrong stt items.